### PR TITLE
add cdt-gdb-vscode

### PR DESCRIPTION
**Pull request by marcdumais-work**
License: EPL-2.0

Signed-off-by: Marc Dumais <marc.dumais@ericsson.com>

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -388,6 +388,12 @@
       "download": "https://github.com/eamodio/vscode-gitlens/releases/download/v11.2.1/gitlens-11.2.1.vsix",
       "version": "11.2.1"
     },
+    {
+      "id": "eclipse-cdt.cdt-gdb-vscode",
+      "repository": "https://github.com/eclipse-cdt/cdt-gdb-vscode",
+      "version": "0.0.91",
+      "checkout": "master"
```
Comment (jankeromnes) [inline extensions.json:395]: Hmm, the script didn't find a better Git ref than `master` for this version. Maybe let's skip `"version"` and just always publish from the default branch:

```suggestion
      "repository": "https://github.com/eclipse-cdt/cdt-gdb-vscode"
```

**Review by jankeromnes — APPROVED** [review]
Looks good to me, thanks @marcdumais-work!

> For this extension, running `yarn` returns an error but the extension looks to work anyway, if one proceeds to package it, e.g. `vsce package`. This could be a problem here i think - the bot will likely stop when `yarn` returns status code 1.

Aha, thanks for the heads up. I think it's fine to try by merging this now and checking back after the CI had a go.

**Comment by jankeromnes** [comment]
@marcdumais-work Oops, auto-publishing failed with:

```
[FAIL] Could not process extension: {
  "id": "eclipse-cdt.cdt-gdb-vscode",
  "repository": "https://github.com/eclipse-cdt/cdt-gdb-vscode"
}
Error: Extension eclipse-cdt.cdt-gdb-vscode version 0.0.91 is already published, but is currently inactive and therefore not visible.
```

@spoenemann is this related to the publisher not having signed the Eclipse thing? What can be done in this case?

**Comment by spoenemann** [comment]
I deleted the previously published extension.

**Comment by jankeromnes** [comment]
Thanks! Auto-publishing now works again 🎉 https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode

**Pull request merged**